Xamarin.Essentials/DeviceDisplay/DeviceDisplay.tizen.cs

```diff
@@ -0,0 +1,100 @@
+﻿using System;
+using System.Runtime.InteropServices;
+using ElmSharp;
+
+namespace Xamarin.Essentials
+{
+    public static partial class DeviceDisplay
+    {
+        [DllImport("libcapi-system-device.so.0", EntryPoint = "device_power_request_lock", CallingConvention = CallingConvention.Cdecl)]
+        internal static extern void ScreenLockDevicePowerRequestLock(int type, int timeout);
+
+        [DllImport("libcapi-system-device.so.0", EntryPoint = "device_power_release_lock", CallingConvention = CallingConvention.Cdecl)]
+        internal static extern void ScreenLockDevicePowerReleaseLock(int type);
+
+        static bool screenLock = false;
+
+        static bool PlatformKeepScreenOn
+        {
+            get
+            {
+                if (!screenLock)
+                    ScreenLockDevicePowerReleaseLock(1);
```

Comment:
What is this `1`? Maybe we should make it a descriptive const?

Reply:
`1` is used instead of an unmanaged enum value. I will change to a descriptive const.